go.mod

```diff
@@ -12,7 +12,7 @@ require (
 	github.com/envoyproxy/go-control-plane v0.9.10-0.20210907150352-cf90f659a021
 	github.com/frankban/quicktest v1.13.1 // indirect
 	github.com/gardener/component-spec/bindings-go v0.0.33
-	github.com/gardener/dependency-watchdog v0.6.1-0.20210623112844-96f73d5dc311
+	github.com/gardener/dependency-watchdog v0.7.0
```

Comment:
Can you also update the version in `charts/images.yaml`?

Reply:
Yes fixed with the [commit](https://github.com/gardener/gardener/pull/5497/commits/2d60faf78e7644cddd6a3b424316b629a7ed483b)